yellowbrick/utils.py

```diff
@@ -155,6 +157,45 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+#From here: http://stackoverflow.com/questions/26248654/numpy-return-0-with-divide-by-zero
+def numpy_div0( a, b ):
+    """
+    Ufunc-extension that returns 0 instead of nan when dividing numpy arrays
+    example: >> div0( [-1, 0, 1], 0 ) -> [0, 0, 0]
+    """
+    with np.errstate(divide='ignore', invalid='ignore'):
+        c = np.true_divide( a, b )
+        c[ ~ np.isfinite( c )] = 0  # -inf inf NaN
+    return c
+
+
+def find_text_color(base_color, text_colors=("black","white")):
+    '''
+    Takes a background color and returns the appropriate light or dark text color.
+    Users can specify the dark and light text color, or accept the defaults of 'black' and 'white'
+
+    # baseColor: the color of the background, in any valid color specification format
+    # textColors: a tuple of (dark, light) text format. 
+    '''
+
+    #matplotlib cmap exports rgba colors scaled 0-1 instead of standard 255
+    rgb=[]
+    for idx,val in enumerate(base_color):
+        rgb.append(val*255)
```

Comment:
I think this would be more understandable as:

```
# multiply the components of the cmap rgb ([0,1]) by 255 to get standard scale. 
rbg = tuple(map(lambda c: c * 255, base_color))
```

Potentially you could also simply divide the constants starting on line 187 by 255.0 and not worry about performing this mapping.